Description: adjust the version check for HTTP::Headers
Origin: vendor
Bug-Debian: http://bugs.debian.org/710873
Forwarded: no
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2013-06-03

--- a/t/api/err_headers_out.t
+++ b/t/api/err_headers_out.t
@@ -5,6 +5,8 @@
 use Apache::TestRequest;
 use Apache::TestUtil;
 
+use version;
+
 plan tests => 6, need 'HTML::HeadParser';
 
 my $location = '/TestAPI__err_headers_out';
@@ -25,7 +27,7 @@
     local $HTTP::Headers::TRANSLATE_UNDERSCORE=
 	$HTTP::Headers::TRANSLATE_UNDERSCORE;
     undef $HTTP::Headers::TRANSLATE_UNDERSCORE
-	if defined HTTP::Headers->VERSION and HTTP::Headers->VERSION==6.00;
+	if defined HTTP::Headers->VERSION and version->parse(HTTP::Headers->VERSION) >= version->parse(6.00);
 
     ok t_cmp $res->header('X-err_headers_out'), "err_headers_out",
         "X-err_headers_out: made it";
@@ -50,7 +52,7 @@
     local $HTTP::Headers::TRANSLATE_UNDERSCORE=
 	$HTTP::Headers::TRANSLATE_UNDERSCORE;
     undef $HTTP::Headers::TRANSLATE_UNDERSCORE
-	if defined HTTP::Headers->VERSION and HTTP::Headers->VERSION==6.00;
+	if defined HTTP::Headers->VERSION and version->parse(HTTP::Headers->VERSION) >= version->parse(6.00);
 
     ok t_cmp $res->header('X-err_headers_out'), "err_headers_out",
         "X-err_headers_out: made it";
