--- ./qt-fsarchiver/qt-fsarchiver.pro
+++ ./qt-fsarchiver-raw/qt-fsarchiver.pro
@@ -56,7 +56,7 @@
 # install
  target.path = /usr/sbin
  icon.files = src/images/harddrive2.png
- icon.path = /usr/share/app-install/icons
+ icon.path = /usr/share/qt-fsarchiver/icons
  autostart.files = starter/qt-fsarchiver.desktop
  autostart.path = /usr/share/applications
  doc.files = doc



--- ./qt-fsarchiver/starter/qt-fsarchiver.desktop
+++ ./qt-fsarchiver/starter/qt-fsarchiver.desktop
@@ -3,7 +3,7 @@
 Name[de_DE]=qt-fsarchiver
 Comment=Qt GUI for fsarchiver
 Exec=/usr/sbin/qt-fsarchiver
-Icon=/usr/share/app-install/icons/harddrive2.png
+Icon=/usr/share/qt-fsarchiver/icons/harddrive2.png
 Terminal=false
 Type=Application
 StartupNotify=true
+Categories=System;

