diff -uNr a/tools/Makefile b/tools/Makefile
--- a/tools/Makefile	2012-03-10 07:19:16.000000000 +0100
+++ b/tools/Makefile	2012-03-12 17:19:04.000000000 +0100
@@ -118,11 +118,17 @@
 		$(GIT) reset --hard $(QEMU_TAG); \
 	fi
 
-subdir-all-qemu-xen-traditional-dir subdir-install-qemu-xen-traditional-dir: qemu-xen-traditional-dir-find
+subdir-all-qemu-xen-traditional-dir: qemu-xen-traditional-dir-find
 	set -e; \
 		$(buildmakevars2shellvars); \
 		cd qemu-xen-traditional-dir; \
 		$(QEMU_ROOT)/xen-setup $(IOEMU_CONFIGURE_CROSS); \
+		$(MAKE)
+
+subdir-install-qemu-xen-traditional-dir: qemu-xen-traditional-dir-find
+	set -e; \
+		$(buildmakevars2shellvars); \
+		cd qemu-xen-traditional-dir; \
 		$(MAKE) install
 
 subdir-clean-qemu-xen-traditional-dir:
@@ -140,14 +146,14 @@
 		$(GIT) reset --hard $(QEMU_UPSTREAM_REVISION); \
 	fi
 
-subdir-all-qemu-xen-dir subdir-install-qemu-xen-dir: qemu-xen-dir-find
+subdir-all-qemu-xen-dir: qemu-xen-dir-find
 	if test -d $(QEMU_UPSTREAM_URL) ; then \
 		source=$(QEMU_UPSTREAM_URL); \
 	else \
 		source=.; \
 	fi; \
 	cd qemu-xen-dir; \
-	$$source/configure --enable-xen --target-list=i386-softmmu \
+	CFLAGS="" $$source/configure --enable-xen --target-list=i386-softmmu \
 		--source-path=$$source \
 		--extra-cflags="-I$(XEN_ROOT)/tools/include \
 		-I$(XEN_ROOT)/tools/libxc \
@@ -156,8 +162,16 @@
 		-L$(XEN_ROOT)/tools/xenstore" \
 		--bindir=$(LIBEXEC) \
 		--disable-kvm \
+		--disable-docs \
+		--datadir=$(DESTDIR)/usr/share/xen/qemu-upstream \
+		--sysconfdir=$(DESTDIR)/etc/xen/qemu-upstream \
 		--python=$(PYTHON) \
 		$(IOEMU_CONFIGURE_CROSS); \
+	sed -i 's#$$(sysconfdir)/qemu#$$(sysconfdir)#' Makefile; \
+	$(MAKE)
+
+subdir-install-qemu-xen-dir: qemu-xen-dir-find
+	cd qemu-xen-dir; \
 	$(MAKE) install
 
 subdir-clean-qemu-xen-dir:
