diff -ur httpd24-orig/t/response/TestModules/proxy.pm httpd24/t/response/TestModules/proxy.pm
--- httpd24-orig/t/response/TestModules/proxy.pm	2013-06-21 13:11:41.421816434 +0000
+++ httpd24/t/response/TestModules/proxy.pm	2013-06-21 13:14:00.011312328 +0000
@@ -47,9 +47,7 @@
     <IfModule mod_proxy.c>
         <Proxy http://@servername@:@port@/*>
             <IfModule @ACCESS_MODULE@>
-                Order Deny,Allow
-                Deny from all
-                Allow from @servername@
+                Require host @servername@
             </IfModule>
         </Proxy>
 
diff -ur httpd24-orig/t/response/TestAPI/access2.pm httpd24/t/response/TestAPI/access2.pm
--- httpd24-orig/t/response/TestAPI/access2.pm	2013-06-21 17:52:12.980765956 +0000
+++ httpd24/t/response/TestAPI/access2.pm	2013-06-21 17:54:54.670189526 +0000
@@ -103,16 +103,17 @@
 
     <IfModule @ACCESS_MODULE@>
         # needed to test $r->satisfies
-        Allow from All
+        Require all granted
     </IfModule>
     AuthType Basic
     AuthName "Access"
-    Require my-user goo bar
-    Require my-group xar tar
-    <Limit POST>
-       Require valid-user
-    </Limit>
-    Satisfy All
+    <RequireAll>
+        Require my-user goo bar
+        Require my-group xar tar
+        <Limit POST>
+            Require valid-user
+        </Limit>
+    </RequireAll>
     <IfModule @AUTH_MODULE@>
         # htpasswd -mbc auth-users goo foo
         # htpasswd -mb auth-users bar mar
diff -ur httpd24-orig/t/filter/TestFilter/both_str_req_proxy.pm httpd24/t/filter/TestFilter/both_str_req_proxy.pm
--- httpd24-orig/t/filter/TestFilter/both_str_req_proxy.pm	2013-06-21 17:59:43.869157727 +0000
+++ httpd24/t/filter/TestFilter/both_str_req_proxy.pm	2013-06-21 18:01:04.218870886 +0000
@@ -62,9 +62,7 @@
     <IfModule mod_proxy.c>
         <Proxy http://@servername@:@port@/*>
             <IfModule @ACCESS_MODULE@>
-                Order Deny,Allow
-                Deny from all
-                Allow from @servername@
+                Require host @servername@
             </IfModule>
         </Proxy>
 
diff -ur httpd24-orig/t/protocol/TestProtocol/pseudo_http.pm httpd24/t/protocol/TestProtocol/pseudo_http.pm
--- httpd24-orig/t/protocol/TestProtocol/pseudo_http.pm	2013-06-21 17:59:43.869157727 +0000
+++ httpd24/t/protocol/TestProtocol/pseudo_http.pm	2013-06-21 18:05:32.087914088 +0000
@@ -154,21 +154,21 @@
 
   <Location TestProtocol::pseudo_http>
 
-      <IfModule @ACCESS_MODULE@>
-          Order Deny,Allow
-          Allow from @servername@
-      </IfModule>
-
-      <IfModule @AUTH_MODULE@>
-          # htpasswd -mbc basic-auth stas foobar
-          # using md5 password so it'll work on win32 too
-          AuthUserFile @ServerRoot@/htdocs/protocols/basic-auth
-      </IfModule>
-
-      AuthName TestProtocol::pseudo_http
-      AuthType Basic
-      Require user stas
-      Satisfy any
+      <RequireAny>
+        <IfModule @ACCESS_MODULE@>
+            Require host @servername@
+        </IfModule>
+
+        <IfModule @AUTH_MODULE@>
+            # htpasswd -mbc basic-auth stas foobar
+            # using md5 password so it'll work on win32 too
+            AuthUserFile @ServerRoot@/htdocs/protocols/basic-auth
+        </IfModule>
+
+        AuthName TestProtocol::pseudo_http
+        AuthType Basic
+        Require user stas
+      </RequireAny>
 
   </Location>
 
