diff --git a/cloudprint/cloudprint.py b/cloudprint/cloudprint.py
index 1c4a4b3..a72d650 100755
--- a/cloudprint/cloudprint.py
+++ b/cloudprint/cloudprint.py
@@ -27,15 +27,17 @@ class CloudPrintProxy(object):
         self.printer_id = None
         self.cups= cups.Connection()
         self.proxy =  platform.node() + '-Armooo-PrintProxy'
-        self.auth_path = os.path.expanduser('~/.cloudprintauth')
+        self.auth_path = '/var/lib/cloudprint/auth'
 
-    def get_auth(self):
+    def get_auth(self,setup=False):
         if self.auth:
             return self.auth
         if not self.auth:
             auth = self.get_saved_auth()
             if auth:
                 return auth
+	    elif setup == False:
+		raise Exception( 'Please generate an auth file using /usr/lib/bin/cloudprint/cloudprint.py -s' )
 
             username = raw_input('Google username: ')
             password = getpass.getpass()
@@ -62,6 +64,8 @@ class CloudPrintProxy(object):
                         'https://www.google.com/accounts/IssuedAuthSubTokens '
                         'to generate an application-specific password.'
                     )
+		else:
+		   raise Exception ( e.msg  )
 
             self.set_auth(auth_response['Auth'])
             return self.auth
@@ -311,16 +315,18 @@ def process_jobs(cups_connection, cpp, printers):
         time.sleep(60)
 
 def usage():
-    print sys.argv[0] + ' [-d][-l][-h] [-p pid_file]'
+    print sys.argv[0] + ' [-d][-l][-h][-s] [-p pid_file]'
     print '-d\t\t: enable daemon mode (requires the daemon module)'
     print '-l\t\t: logout of the google account'
     print '-p pid_file\t: path to write the pid to (default cloudprint.pid)'
+    print '-s\t\t: Setup google account'
     print '-h\t\t: display this help'
 
 def main():
-    opts, args = getopt.getopt(sys.argv[1:], 'dlhp:')
+    opts, args = getopt.getopt(sys.argv[1:], 'sdlhp:')
     daemon = False
     logout = False
+    setup = False
     pidfile = None
     for o, a in opts:
         if o == '-d':
@@ -329,6 +335,8 @@ def main():
             logout = True
         elif o == '-p':
             pidfile = a
+	elif o == '-s':
+	    setup = True
         elif o =='-h':
             usage()
             sys.exit()
@@ -338,6 +346,10 @@ def main():
     cups_connection = cups.Connection()
     cpp = CloudPrintProxy()
 
+    if setup:
+	cpp.get_auth(setup)
+	return
+
     if logout:
         cpp.del_saved_auth()
         print 'logged out'
